swablu/config.py

```diff
@@ -153,6 +153,13 @@ def get_jam(dbcon, key):
     return json.loads(d['config'])
 
 
+def jam_exists(dbcon, key):
+    cursor = db_cursor(dbcon, dictionary=True, buffered=True)
+    sql = f"SELECT * FROM `{TABLE_NAME_JAM}` WHERE `key` = %s"
```

Comment:
Can use "SELECT 1" instead of "SELECT *" to reduce data transferred, since you're only checking for the existance of a row.

Reply:
Wouldn't that be `SELECT * [...] LIMIT 1`? Or is the syntax different here?

Reply:
The body of a select statement doesn't need to select columns, selecting any constant works too. `SELECT 1 FROM Table` returns a result set with one column containing the value 1 for every row in the table. Combine with limit 1 for even more performance.

Not that this really matters for our purposes where there's so little data that indexes wouldn't kick in if we had any.

Reply:
Oh, right, I remember hearing about that now. Probably not worth it anyway.